### PR TITLE
Issue 1388 lower level remove unnecessary logging

### LINKED ISSUE #1388: Lower level or remove unnecessary logging
Thank you for contributing to this project!

- Make sure to search the [issues](https://github.com/tl-its-umich-edu/my-learning-analytics/issues) for duplicates first!

### Describe the bug (Tell us what happens instead of the expected behavior) :
In the logs there are many unnecessary and not useful messages printed to the INFO log level

These include:
```
[2022-06-29 09:59:58 -0400] [INFO] [utils.py:70] course_id from path: #######
[2022-06-29 09:58:40 -0400] [INFO] [views.py:549] user XXXXX in course XXXXXXXX don't have any defaults values set type resource
[2022-06-29 09:58:46 -0400] [INFO] [views.py:289] course_id XXXXX total student=XX
[2022-06-29 10:00:16 -0400] [INFO] [db_util.py:146] User XXXXXXX is enrolled in these courses:
[2022-06-29 10:03:46 -0400] [INFO] [views.py:538] get_user_default_selection
[2022-06-29 10:03:46 -0400] [INFO] [views.py:776] get_course_date_start
[2022-06-29 10:05:05 -0400] [INFO] [db_util.py:116] get_user_courses_info
[2022-06-29 12:18:01 -0400] [INFO] [lti_new.py:103] LTI configuration valid.
[2022-06-29 09:42:31 -0400] [INFO] [views.py:167] update_course_info
```

These all _seem_ like debug messages to me. Add any others to this list. This is somewhat related to cleaning up #1381 and also #228

You can avoid changing cron.py as we are mostly only concerned about the webapp.

### PR DESCRIPTION
**Pull request by jxiao21**
Fixes #1388 

Still need feedback on which INFO level statements need to be lowered to DEBUG

### CODE REVIEW

**Comment by jonespm** [comment]
The only comments I have are around cron. I think we still want to print all of those out to be able to see as the cron is running where it is in the process.

I was thinking at one time of having a decorator clean this up a little, maybe you could implement that?
Something like this
```
def log_function_call(func):
    @wraps(func)
    def wrapper(*args, **kwargs):
        logging.info(f"Calling function: {func.__name__}")
        result = func(*args, **kwargs)
        logging.info(f"Function {func.__name__} completed")
        return result
    return wrapper
```

Then remove all of these logging and just put

`@log_function_call
`
in front of each of these functions? I think that would be useful and cleaner than what we have here. Everything outside the cron looks good to me!

**Comment by jxiao21** [comment]
Would `@log_function_call` go in front of every function, or just in front of the ones where we originally had the logging.info('function call') statements? Also, would those logging statements just be removed, or changed back to logging.info?

File: dashboard/cron.py [hunk 1]
```diff
@@ -23,6 +23,16 @@
 
 logger = logging.getLogger(__name__)
 
+# Decorator to clean up function call logging
+def log_function_call(func):
+    @wraps(func)
```
Comment (zqian) [inline dashboard/cron.py:28]: I got "name 'wraps' is not defined" error. It is fixed by adding the import line `from functools import wraps`

`Running Crons
========================================
Traceback (most recent call last):
  File "/code/manage.py", line 12, in <module>
    execute_from_command_line(sys.argv)
  File "/usr/local/lib/python3.10/site-packages/django/core/management/__init__.py", line 442, in execute_from_command_line
    utility.execute()
  File "/usr/local/lib/python3.10/site-packages/django/core/management/__init__.py", line 436, in execute
    self.fetch_command(subcommand).run_from_argv(self.argv)
  File "/usr/local/lib/python3.10/site-packages/django/core/management/base.py", line 412, in run_from_argv
    self.execute(*args, **cmd_options)
  File "/usr/local/lib/python3.10/site-packages/django/core/management/base.py", line 458, in execute
    output = self.handle(*args, **options)
  File "/usr/local/lib/python3.10/site-packages/django_cron/management/commands/runcrons.py", line 45, in handle
    crons_to_run = [get_class(x) for x in cron_class_names]
  File "/usr/local/lib/python3.10/site-packages/django_cron/management/commands/runcrons.py", line 45, in <listcomp>
    crons_to_run = [get_class(x) for x in cron_class_names]
  File "/usr/local/lib/python3.10/site-packages/django_cron/__init__.py", line 33, in get_class
    m = __import__(module)
  File "/code/dashboard/cron.py", line 37, in <module>
    class DashboardCronJob(CronJobBase):
  File "/code/dashboard/cron.py", line 159, in DashboardCronJob
    def verify_course_ids(self):
  File "/code/dashboard/cron.py", line 28, in log_function_call
    @wraps(func)
NameError: name 'wraps' is not defined`

Comment (zqian) [reply dashboard/cron.py:28]: @jxiao21 Could you please verify whether this is still an issue? Thanks!

Comment (jxiao21) [reply dashboard/cron.py:30]: I added the import and pushed it

**Review by zqian — APPROVED** [review]
Tested and the log messages are formatted as expected.

**Pull request merged**